content/en/monitors/monitor_types/forecasts.md

```diff
@@ -15,91 +15,126 @@ further_reading:
-
 {{< img src="monitors/monitor_types/forecasts/linear_reactive.png" alt="linear reactive" responsive="true" style="width:80%;" >}}
 
-The "default" model is the middle of the road choice which adjusts to the most recent trend. It extrapolates a line while being robust to recent noise.
-
-{{< img src="monitors/monitor_types/forecasts/linear_default.png" alt="linear default" responsive="true" style="width:80%;">}}
+{{% /tab %}}
+{{% tab "Seasonal" %}}
 
-**Seasonal:** Use this algorithm for seasonal metrics. In monitors, Datadog auto-detects the seasonality of the metric and chooses between weekly, daily, and hourly seasonality. This algorithm requires at least two seasons of history for it to start forecasting, and potentially uses up to six.
+Use the seasonal algorithm for metrics with repeating patterns. There are four different _seasonality_ choices:
 
-Examples of seasonality options:
+| Option  | Description                                                                                                                                        |
+|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Hourly  | The algorithm expects the same minute after the hour behaves like past minutes after the hour, for example 5:15 behaves like 4:15, 3:15, etc.      |
+| Daily   | The algorithm expects the same time today behaves like past days, for example 5pm today behaves like 5pm yesterday.                                |
+| Weekly  | The algorithm expects that a given day of the week behaves like past days of the week, for example this Tuesday behaves like past Tuesdays.        |
+| Monthly | The algorithm expects that a given day of the month behaves like past days of the month, for example July 1st behaves like June 1st, May 1st, etc. |
 
-* **weekly**: the algorithm expects this Monday behaves like past Mondays.
-* **daily**: the algorithm assumes that 7 pm today is like 7 pm from previous days.
-* **hourly**: the algorithm expects that 7:15 behaves like 6:15, 5:15, 4:15, etc.
+**Note**: This algorithm requires at least two seasons of history and uses up to six seasons for forecasting.
 
 {{< img src="monitors/monitor_types/forecasts/seasonal.png" alt="seasonal" responsive="true" style="width:80%;">}}
 
-### Accessing Advanced Options
+{{% /tab %}}
+{{< /tabs >}}
+
+### Notifications
+
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][6] page.
+
+## API
+
+To create forecast monitors programmatically, see the examples below and the [Datadog API reference][7].
+
+{{< tabs >}}
+{{% tab "Linear" %}}
 
-Access advanced options under the **Advanced** tab in the **New Monitor** page. To specify them in the dashboards (using the JSON tab) or in the API, use this format:
+```text
```

Comment:
Like above, I think we should strongly encourage the usage of Export Monitor instead of manually building queries.